### PR TITLE
Remove thekindkeptn

### PR DESCRIPTION
**Pull request by agardnerIT**
Removes thekindkeptn from docs since we have zero-install option now.

Signed-off-by: agardnerit <adam@agardner.net>

### CODE REVIEW

**Review by StackScribe — APPROVED** [review]
How I wish we could just drive a stake through this QuickStart Guide -- and then maybe rename "Explore Keptn" to be Quick Start, since that seems to be the title people are looking for.  Ah well.

File: content/docs/quickstart/_index.md [hunk 1]
```diff
@@ -120,20 +84,34 @@ curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v4.4
...
 
-2. **Download and install the [Keptn CLI](../0.14.x/reference/cli)**
+2. **Download and install the [Keptn CLI](../0.17.x/reference/cli)**
 
     ```
     curl -sL https://get.keptn.sh | bash
     ```
 
-3. **Install Keptn** control-plane and execution-plane for continuous delivery use case or use the `helm install` version [mentioned below](#kubernetes-version-not-supported).
+3. Install core the control plane components and expose via a LoadBalancer:
```
Comment (agardnerIT) [inline content/docs/quickstart/_index.md:93]: `Install the core control plane components` ?

**Review by StackScribe — DISMISSED** [review]
Whoops, another broken xref

**Comment by agardnerIT** [comment]
Merge please?

**Comment by agardnerIT** [comment]
I clearly don't know git well enough. I've totally messed this branch up thanks to DCO.

Someone please help to fix. At this point I'm doing more damage than good.

**Comment by agardnerIT** [comment]
@johannes-b can you review please? Last thekindkeptn build was `0.16.0` so quite old.

**Comment by agardnerIT** [comment]
@StackScribe I've updated doc links to 0.18. Please merge. Once done I'll raise another PR to remove the first FAQ entry as it mentions `keptn install` which is no longer supported. I didn't want to do two changes inside this PR.

**Comment by agardnerIT** [comment]
@StackScribe fixed all links and overrode the DCO as it provides zero value.

**Review by oleg-nenashev — APPROVED** [review]
+1, thanks!

**Pull request merged**